library/src/main/java/com/chuckerteam/chucker/internal/support/ShareUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.chuckerteam.chucker.internal.support
+
+import android.content.Context
+import com.chuckerteam.chucker.R
+import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
+
+internal object ShareUtils {
+    fun getStringFromTransactions(transactions: List<HttpTransaction>, context: Context): String {
```

Comment:
I think that this fun might have to deal with pretty big lists of transactions. Let's turn it into suspend fun as well to something like this:
```suggestion
     suspend fun getStringFromTransactions(transactions: List<HttpTransaction>, context: Context): String =
        withContext(Dispatchers.Default) {
            return@withContext transactions.joinToString(
                separator = "\n${context.getString(R.string.chucker_export_separator)}\n",
                prefix = "${context.getString(R.string.chucker_export_prefix)}\n",
                postfix = "\n${context.getString(R.string.chucker_export_postfix)}\n"
            ) { FormatUtils.getShareText(context, it, false) }
        }
```

Reply:
updated.